jicofo/rootfs/defaults/jicofo.conf

```diff
@@ -24,6 +24,8 @@
 {{ $XMPP_RECORDER_DOMAIN := .Env.XMPP_RECORDER_DOMAIN | default "recorder.meet.jitsi" -}}
 {{ $XMPP_PORT := .Env.XMPP_PORT | default "5222" -}}
 {{ $XMPP_SERVER := .Env.XMPP_SERVER | default "xmpp.meet.jitsi" -}}
+{{ $MAX-SSRCS-PER-USER := .Env.MAX-SSRCS-PER-USER | default "20" -}}
+{{ $MAX-SSRC-GROUPS-PER-USER := .Env.MAX-SSRC-GROUPS-PER-USER | default $MAX-SSRCS-PER-USER -}}
```

Comment:
What's the default in the JVB?

Reply:
@saghul    default value is 20 in jicofo

<img width="848" alt="image" src="https://github.com/jitsi/docker-jitsi-meet/assets/7316187/34127479-1b6d-4df9-8f35-bc382da346a4">